common/scala/src/main/scala/whisk/utils/ExecutionContextFactory.scala

```diff
@@ -17,22 +17,36 @@
 package whisk.utils
 
 import java.util.concurrent.ArrayBlockingQueue
+import java.util.concurrent.Executors
 import java.util.concurrent.ThreadPoolExecutor
+
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
+import scala.concurrent.Promise
 import scala.concurrent.duration.Duration
 import scala.concurrent.duration.DurationInt
 import scala.concurrent.duration.FiniteDuration
-import akka.actor.ActorSystem
-import akka.pattern.after
-import java.util.concurrent.Executors
 import scala.language.postfixOps
 
+import akka.actor.ActorSystem
+import akka.pattern.{ after => expire }
```

Comment:
renamed `after` to `expire` to reuse `after` below for better readability on call side.